src/new-components/image/image.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+import ExampleImage from '../../images/office/sy-office-01.jpg';
+import { Image } from './image';
+import styled from 'styled-components';
+
+const SampleImage = styled.img`
+  width: 100%;
+`;
+
+const Template: ComponentStory<typeof Image> = (args) => (
+  <Image {...args}>
+    <SampleImage alt="" src={ExampleImage} />
+  </Image>
+);
+
+export const Regular = Template.bind({});
+Regular.args = {
+  textAlign: 'right',
+  description:
+    'Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Morbi leo risus, porta ac consectetur ac, vestibulum at eros. ',
+  attribution: {
+    source: 'https://www.example.com',
+    creator: 'Banksy',
+  },
+};
```

Comment:
Let's only use the `mdx` format for documentation. You can define this story in the `stories.mdx` file so we can completely remove the `stories.tsx` file.

Reply:
oh snap, you are right I missed this